.eslintrc.json

```diff
@@ -10,7 +10,9 @@
     "react/jsx-props-no-spreading": 0,
     "react/jsx-one-expression-per-line": 0,
     "class-methods-use-this": 0,
-    "import/prefer-default-export": 0
+    "import/prefer-default-export": 0,
+    "no-useless-return": 0,
+    "consistent-return": 0
```

Comment:
The reason for adding it is as follows.

We're already getting help from typescript, so I think the above configuration is superfluous.🤔

### no-useless-return

![image](https://user-images.githubusercontent.com/10924072/121775300-b7efba00-cbc1-11eb-9c88-0d750433f9a5.png)

If the return statement is deleted, the following error occurs
![image](https://user-images.githubusercontent.com/10924072/121775354-0b620800-cbc2-11eb-89d4-bc0778d018a0.png)


### consistent-return
![image](https://user-images.githubusercontent.com/10924072/121775282-a9a19e00-cbc1-11eb-8637-8c2f32724ee3.png)